### PR TITLE
Dropping NormalizedURI from ScraperURISummaryCommanderImpl

### PR DESCRIPTION
**Pull request by andrewconner**
`ScraperURISummaryCommander` uses NormalizedURI a ton, only needing `id`, `externalId`, and `url`. Since these parts are a bit hairy, doing them separately.

The risk here is if a client is ever currently sending a NormalizedURI without an id. We'll get serialization issues if that happens.

@stkem

### CODE REVIEW

**Comment by andrewconner** [comment]
Mind going over this? I'd like to merge/deploy to watch it, though.

File: kifi-backend/modules/shoebox/test/com/keepit/commanders/URISummaryCommanderTest.scala [hunk 1]
```diff
@@ -58,16 +56,13 @@ class URISummaryCommanderTestImageFetcher extends ImageFetcher {
 }
 
 case class URISummaryCommanderTestS3URIImageStore() extends S3URIImageStore {
-  def storeImage(info: ImageInfo, rawImage: BufferedImage, nUri: NormalizedURI): Try[(String, Int)] = Success((FakeS3URIImageStore.placeholderImageURL, FakeS3URIImageStore.placeholderSize))
-  def getDefaultScreenshotURL(nUri: NormalizedURI): Option[String] = Some(FakeS3URIImageStore.placeholderScreenshotURL)
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/test/com/keepit/commanders/URISummaryCommanderTest.scala:62]: So much of this wasn't even being used.

File: kifi-backend/modules/scraper/app/com/keepit/commanders/ScraperURISummaryCommander.scala [hunk 2]
```diff
@@ -1,10 +1,12 @@
 package com.keepit.commanders
```
Comment (andrewconner) [inline kifi-backend/modules/scraper/app/com/keepit/commanders/ScraperURISummaryCommander.scala:1]: todo: rename to `com.keepit.scraper.commanders`

**Comment by yingjieMiao** [comment]
lgtm

**Comment by stkem** [comment]
lgtm

**Pull request merged**